datafusion/sql/src/parser.rs

```diff
@@ -134,7 +134,7 @@ impl<'a> DFParser<'a> {
     /// Parse a sql string into one or [`Statement`]s using the
     /// [`GenericDialect`].
     pub fn parse_sql(sql: &str) -> Result<VecDeque<Statement>, ParserError> {
-        let dialect = &GenericDialect {};
+        let dialect = &PostgreSqlDialect {};
```

Comment:
I am unsure about the best approach to tackle this. One potential solution is to provide a configuration option that users can set at the beginning.